pages/index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css"
+    />
+    <link href="https://asthra-ui.netlify.app/style.css" rel="stylesheet">
+    <link href="/css/home.css" rel="stylesheet">
+</head>
+<body>
+    <nav class = "nav-bar">
+        <span class ="nav-title">BakeHere</span>
+        <div class = "input-box-icon">
+            <label for = "input-search"></label>
+            <div class = "input-icon-container border-grey">
+                <button><i class="bi bi-search"></i></button>
+                <input type = "text" id ="input-search" placeholder="Search">
+            </div>
+        </div>
+        <div class = "btn-nav">
+            <button class="btn-login">
+              <a href = "/" >Login</a> 
+            </button>
+            <button class ="btn-icon">
+                <a href="/"><i class="bi bi-heart-fill"></i></a>
+            </button>
+            <button class ="btn-icon">
+                <a href="/"><i class="bi bi-cart-fill"></i></a>
+            </button>
+        </div>
+    </nav>
+    <div class = "home-pg-container">
+       
```

Comment:
consider removing extra spaces

Reply:
Thanks for pointing that out. I had removed that.